src/QueueBatch/Impl/BindingProvider.cs

```diff
@@ -51,10 +56,15 @@ public async Task<ITriggerBinding> TryCreateAsync(TriggerBindingProviderContext
 
             return new TriggerBinding(context.Parameter, queue, TimeSpan.FromSeconds(attr.MaxBackOffInSeconds), attr.ParallelGets, attr.RunWithEmptyBatch, loggerFactory);
         }
-
         CloudQueue CreatePoisonQueue(CloudQueueClient queueClient, string name)
         {
             return queueClient.GetQueueReference(name + PoisonQueueSuffix);
         }
+
+        string ResolveName(string queueName)
+        {
+            var resolvedName = nameResolver?.ResolveWholeString(queueName) ?? queueName;
+            return resolvedName.ToLowerInvariant();
```

Comment:
I needed to double check whether the queue name is case invariant. It is https://blogs.msdn.microsoft.com/jmstall/2014/06/12/azure-storage-naming-rules/